### PR TITLE
More document for meter configurations in backend-meter.md

### PR DESCRIPTION
**Pull request by sialais**
Refine backend-meter.md. Adding description on meter configurations.

### CODE REVIEW

File: docs/en/setup/backend/backend-meter.md [hunk 1]
```diff
@@ -42,9 +58,10 @@ expSuffix: <string>
 metricPrefix: <string>
 # Metrics rule allow you to recompute queries.
 metricsRules:
-  # The name of rule, which combinates with a prefix 'meter_' as the index/table name in storage.
+  # The name of rule, which combined with a prefix '<metricPrefix>_' as the index/table name in storage.
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-meter.md:61]:
```suggestion
  # The name of rule, which combinates with a prefix '<metricPrefix>_' as the index/table name in storage.
```

File: docs/en/setup/backend/backend-meter.md [hunk 2]
```diff
@@ -17,14 +18,29 @@ kafka-fetcher:
     bootstrapServers: ${SW_KAFKA_FETCHER_SERVERS:localhost:9092}
 ```
 
+## Meter collection
+There are plentiful metrics are collected by default, you may find predefined metrics by oal files located in `config/oal/`.
```
Comment (wu-sheng) [inline docs/en/setup/backend/backend-meter.md:22]:
```suggestion
```
I think we should remove this, due to OAL doesn't relate to MAL at all.

**Comment by wu-sheng** [comment]
Please update the changes.md in the root folder

File: docs/en/setup/backend/backend-meter.md (hunk 2 above)
Comment (wu-sheng) [reply docs/en/setup/backend/backend-meter.md:22]: https://skywalking.apache.org/docs/main/latest/en/guides/backend-oal-scripts/#official-oal-script This is the doc for OAL

**Comment by sialais** [comment]
> Please update the changes.md in the root folder

CHANGES.md updated.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**